content/community/contributor-how-tos.md

```diff
@@ -0,0 +1,59 @@
+
+Here is a standard usage pattern that contributors can adopt to specific changes as they see fit:
+
+1. [Download the microplane binary from their releases page](https://github.com/Clever/microplane/releases)
+1. Open your terminal, rename and add the downloaded binary into your $PATH, and add execution privileges to the binary:
+   1. `mv ~/Downloads/mp-0.0.21-darwin-amd64 /usr/local/bin/mp && chmod 755 /usr/local/bin/mp`
+1. Add a [GH Personal Access Token](https://docs.github.com/en/github/authenticating-to-github/creating-a-personal-access-token) to your shell's environment variables under the variable `GITHUB_API_TOKEN`:
+   1. `export GITHUB_API_TOKEN=$YOUR_TOKEN`
+1. Change to an empty directory that you can use as a scratch workspace for your Many Repos change:
+   1. `mkdir ~/mp_workspace && cd ~/mp_workspace`
+1. Initialize microplane:
+   1. `mp init --all-repos "cloudposse"`
+   1. Initializing creates an `mp/` folder in your current directoy, finds all of the Cloud Posse public projects, and then creates an `mp/init.json` file describing them.
+   1. NOTE: microplane supposedly has the ability to search against an organization and narrow the returned repos that end up in `init.json`, but that functionality seems buggy and not working. We do our repo filtering manually in the next step.
+1. Manually edit `mp/init.json` to only include the repos which you want to make changes against.
+   1. Your editor's multi-select and edit capabilities are your friend here!
+   1. Duplicate the original `init.json` so you don't need to re-run `mp init` in the case that you want to start fresh.
+1. Run microplane's clone command to pull all of the repos specified in `init.json` down to your local machine for mass changes:
+   1. `mp clone`
+1. Create a bash script to facilitate the changes that you're attempting to make against the many repos you've specified.
+   1. Use the microplane `-r` or `--repo` flag to operate on a single repo for testing your script prior to making the changes across all repos.
+   1. Go through the full microplane process (complete the following steps) for this single repo test-run and get it signed off by the other Cloud Posse contributors to ensure everyone agrees with the change prior to making it.
+1. Once you've got a script that is working like you expect, you can run the microplane plan command. This step executes the given script across all the repos specified in `init.json` and then commits the result. You should format your `mp plan` command as follows:
+    > mp plan -b $YOUR_BRANCH_NAME -m "[AUTOMATED] $YOUR_PR_TITLE
+    >
+    > ## What
+    >
+    > 1. $INFO_ON_YOUR_CHANGE_NUMBER_ONE
+    > 1. $INFO_ON_YOUR_CHANGE_NUMBER_TWO
+    >
+    > ## Why
+    >
+    > 1. $INFO_ON_WHY_YOU_MADE_YOUR_CHANGE_NUMBER_ONE
+    > 1. $INFO_ON_WHY_YOU_MADE_YOUR_CHANGE_NUMBER_TWO" \
+    > -- \
+    > sh -c $PWD/$YOUR_SCRIPT_NAME.sh
```

Comment:
~~~suggestion
1. Once you've got a script that is working like you expect, you can run the microplane plan command. This step executes the given script across all the repos specified in `init.json` and then commits the result. You should format your `mp plan` command as follows: 
    ```bash
    mp plan -b $YOUR_BRANCH_NAME -m "[AUTOMATED] $YOUR_PR_TITLE
    
    ## What
    
    1. $INFO_ON_YOUR_CHANGE_NUMBER_ONE
    1. $INFO_ON_YOUR_CHANGE_NUMBER_TWO
    
    ## Why
    
    1. $INFO_ON_WHY_YOU_MADE_YOUR_CHANGE_NUMBER_ONE
    1. $INFO_ON_WHY_YOU_MADE_YOUR_CHANGE_NUMBER_TWO" \
    -- \
    sh -c $PWD/$YOUR_SCRIPT_NAME.sh
    ```
    __NOTE__ the quotes around the message passing this as an argument to `-m` (message)
~~~

Reply:
Done.